quickstart.py

```diff
@@ -83,19 +82,24 @@ def missingDependencies():
   missing = []
   try:
     from concurrent.futures import Future
+    assert Future
```

Comment:
pyflakes complains about unused imports. I want to have Travis CI run pyflakes, so I wanted to address all errors raised by it.

Since these import statements exist purely to test if the packages can be imported, I wanted to have the linter ignore those unused imports. An assert statement quiets pyflakes and I feel that it retains the clarity of the code.